### PR TITLE
Remove node-sass

### PR DESCRIPTION
**Pull request by zofskeez**
As part of the sidebar navigation work, the design system components were brought in but I could not include the design system tokens css by using the `includePaths` option, however I was able to get it working as an `app.import`. I tracked this down to using `node-sass` rather than the default `ember-cli-sass` implementation (Dart Sass). To avoid further issues I removed `node-sass` and got the sass compiling with the default implementation.

Initially after making the changes the sass compiler was throwing a syntax error for an undefined variable. I tracked this down to HDS having several of the same named files (empty-state, modal and tabs) and the use of globally scoped variables. To work around this for now I appended _-component_ to these file names but a better long term solution would be to update to `@use` [variables so they are locally scoped](https://sass-lang.com/documentation/at-rules/use). 
image

There are also some deprecation warnings coming from Bulma files which should be fine for now since we are in the process of removing that library.
image

I tested using a token from the design systems css and they are now available.

### CODE REVIEW

File: ui/app/styles/components/tabs-component.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@
       }
     }
     li {
-      > a &.active {
+      a &.active {
```
Comment (zofskeez) [inline ui/app/styles/components/tabs-component.scss:23]: This was coming up as an invalid selector warning so I removed the leading `>`

File: ui/app/styles/utils/_bulma_variables.scss [hunk 2]
```diff
@@ -28,11 +28,11 @@ $family-sans: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto'
...
 $family-primary: $family-sans;
 $body-size: 14px;
-$size-3: (24/14) + 0rem; // ~1.714rem ~27px
+$size-3: calc(24 / 14) + 0rem; // ~1.714rem ~27px
 $size-4: 1.5rem; // 24px
 $size-5: 1.25rem; // 20px
-$size-7: (13/14) + 0rem; // ~.929rem ~15px
-$size-8: (12/14) + 0rem; // ~.857rem  ~13.7px
+$size-7: calc(13 / 14) + 0rem; // ~.929rem ~15px
+$size-8: calc(12 / 14) + 0rem; // ~.857rem  ~13.7px
```
Comment (zofskeez) [inline ui/app/styles/utils/_bulma_variables.scss:35]: Deprecation warnings from Dart Sass with the recommended changes

File: ui/app/styles/components/tabs-component.scss (hunk 1 above)
Comment (zofskeez) [reply ui/app/styles/components/tabs-component.scss:23]: @Monkeychip this came up in a recent PR of yours if I recall correctly. Should I update this further to `a.active`? That fixes the problem.
image

Comment (zofskeez) [reply ui/app/styles/components/tabs-component.scss:23]: Actually, I think what was originally intended was this:
```scss
li {
  > a {
    &.active {
      border-color: $blue;
      color: $blue !important;
    }
  }
}
```

Comment (Monkeychip) [reply ui/app/styles/components/tabs-component.scss:23]: Oh, yeah go for it! Nice find.

File: ui/package.json [hunk 3]
```diff
@@ -190,6 +189,7 @@
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
     "route-recognizer": "^0.3.4",
+    "sass": "^1.58.3",
```
Comment (Monkeychip) [inline ui/package.json:192]: did we want the carrot here or did we want to lock it in?

**Review by Monkeychip — APPROVED** [review]
Looks good to me, only one non-blocking question.

File: ui/package.json (hunk 3 above)
Comment (zofskeez) [reply ui/package.json:192]: This is something that I've wanted to revisit so thanks for the reminder. I'm going to leave as is for now and I've added an item to our next sync.

**Pull request merged**